src/pages/BuildPage/BuildPage.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Redirect, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import './BuildPage.css';
+
+class BuildPage extends Component {
+  render() {
+    const { selectedBuild } = this.props;
+    if (!selectedBuild) {
+      return (
+        <Redirect
+          to='/'
+        />
+      )
+    }
+    let statusClass = 'text-dark';
+    if(selectedBuild.status === 'fixed' || selectedBuild.status === 'success' ) {
```

Comment:
Same as earlier, better use switch/object in cases similar to this, it's faster and also increases readability